src/main/ngapp/accio/src/app/toolbar/toolbar.component.ts

```diff
@@ -12,4 +12,7 @@ export class ToolbarComponent implements OnInit {
   ngOnInit(): void {
   }
 
+  getTolerance(value: number) {
```

Comment:
What is this for?

Reply:
I put off integrating user input for tolerance in the mask toolabar until the MVP is done (ie being able to upload photos, I will add a comment.